tensorflow/buildspec.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK tensorflow
-version: &VERSION 2.5.0
-short_version: &SHORT_VERSION 2.5
+version: &VERSION 2.3.4
```

Comment:
Revert changes in `tensorflow/buildspec.yml` file

Reply:
Also, update the `version` to`2.3.4` in [buildspec-2-3.yml](https://github.com/aws/deep-learning-containers/blob/master/tensorflow/buildspec-2-3.yml)